Ubuntu-20/Dockerfile

```diff
@@ -97,36 +102,41 @@ RUN apt install -y \
       --slave /usr/bin/riscv64-linux-gnu-gcov riscv64-linux-gnu-gcov /usr/bin/riscv64-linux-gnu-gcov-${GCC_MAJOR_VERSION} && \
     update-alternatives \
       --install /usr/bin/riscv64-linux-gnu-cpp riscv64-linux-gnu-cpp /usr/bin/riscv64-linux-gnu-cpp-${GCC_MAJOR_VERSION} 100
+
+# Set toolchains prefix
 ENV GCC5_AARCH64_PREFIX /usr/bin/aarch64-linux-gnu-
 ENV GCC5_ARM_PREFIX     /usr/bin/arm-linux-gnueabi-
 ENV GCC5_RISCV64_PREFIX /usr/bin/riscv64-linux-gnu-
 
-# Install deps.
-RUN apt install -y build-essential uuid-dev git virtualenv \
-    device-tree-compiler mono-devel wget
-
-# Cleanup
-RUN apt-get -y autoremove
-RUN apt-get clean
+# Set the locale
+RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && \
+    locale-gen
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
 
 # Install edkrepo
-RUN mkdir /edkrepo_install && cd /edkrepo_install && \
+RUN mkdir /edkrepo_install && \
+    cd /edkrepo_install && \
     wget -O- ${EDKREPO_URL} | tar zxvf - && \
     ./install.py --no-prompt --user $(id -nu) && \
     mkdir -p /etc/edkrepo_skel && \
     cp -R /root/.edkrepo /etc/edkrepo_skel && \
     rm -rf /edkrepo_install
-COPY init_edkrepo_conf.sh /usr/bin/init_edkrepo_conf
 
+COPY init_edkrepo_conf.sh /usr/bin/init_edkrepo_conf
 
-#####################################################################
 # Dev Image
-#
 FROM build AS dev
 
 # Install convenience tools.  Things we like having around, but aren't
 # required.
-RUN apt-get install -y vim nano bear
+RUN apt-get install --yes --no-install-recommends \
+        vim \
+        nano \
+        bear &&\
+    apt-get clean &&\
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Would be nice to include this cleanup at the end up the build image, too.  I'm more worried about build image size than dev image.  That would mean doing another apt update to start the dev image, though.

Reply:
Yep, I tried to cleanup in the build image before, but in the dev image I needed to re-update. so I decided to cleanup in the dev image, we just make sure that the installed package can be developed properly, if the developer has other needs, do the apt update operation by himself, it's not a troublesome operation.